docs/config.md

```diff
@@ -134,32 +125,32 @@ public OpenTelemetry.Trace.TracerProviderBuilder ConfigureTracerProvider(OpenTel
 ```
 
 The plugin must use the same version of the `OpenTelemetry` as the
-OpenTelemetry .NET Auto-Instrumentation.
+OpenTelemetry .NET Automatic Instrumentation.
 
 ## .NET CLR Profiler
 
 To perform bytecode instrumentation, configure the OpenTelemetry .NET
-auto-instrumentation as a .NET CLR Profiler.
-Below are the environment variables used by CLR to setup the profiler.
+Automatic Instrumentation as a .NET CLR Profiler. The CLR uses the following
+environment variables to set up the profiler.
 
 | Environment variable | Description | Value |
 |-|-|-|
-| `CORECLR_ENABLE_PROFILING` | Enable the profiler. | `1` |
+| `CORECLR_ENABLE_PROFILING` | Enables the profiler. | `1` |
 | `CORECLR_PROFILER` | CLSID of the profiler. | `30000` (ms) |
 | `CORECLR_PROFILER_PATH` | Path to the profiler. | `%InstallationLocation%/OpenTelemetry.AutoInstrumentation.Native.so` for Linux, `%InstallationLocation%/OpenTelemetry.AutoInstrumentation.Native.dylib` for MacOS |
-| `CORECLR_PROFILER_PATH_32` | Path to the 32 bit profiler. | `%InstallationLocation%/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll` for Windows |
-| `CORECLR_PROFILER_PATH_64` | Path to the 64 bit profiler. | `%InstallationLocation%/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll` for Windows |
+| `CORECLR_PROFILER_PATH_32` | Path to the 32-bit profiler. | `%InstallationLocation%/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll` for Windows |
+| `CORECLR_PROFILER_PATH_64` | Path to the 64-bit profiler. | `%InstallationLocation%/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll` for Windows |
 
-Remarks:
+### Considerations on the CLR profiler
 
-1. .NET Framework uses `CLR_` prefix instead of `CORECLR_`.
-2. The bitness-specific variables take precedence.
-3. The `*_PROFILER_PATH_*` environment variable is not needed on Windows
-   if the `.dll` is registered.
+- .NET Framework uses the `CLR_` prefix instead of `CORECLR_`.
```

Comment:
The prefix is actually `COR_`, see https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/profiling/setting-up-a-profiling-environment

Reply:
What about moving this comment about the prefix to some place above the table?

Reply:
Done!